django_pydantic_field/compat/imports.py

```diff
@@ -37,7 +37,7 @@ def _import_compat_module(module_name: str) -> types.ModuleType:
     elif PYDANTIC_V1:
         module_path_parts.append("v1")
     else:
-        raise RuntimeError(f"Pydantic {PYDANTIC_VERSION} is not supported")
```

Comment:
Oops, I didn't know the `PYDANTIC_VERSION` is being used... If this is the only place, it should be fine 😬